src/discordChannelMonitor/DiscordChannelMonitor.js

```diff
@@ -0,0 +1,58 @@
+class DiscordChannelMonitor {
```

Comment:
The class name suggests this is a very generic monitoring class. However, this serves a very specific purpose (the checks are specific to the showcase channel usecase). Can we rename it to fit that?